web-components/src/components/phone-input/PhoneInput.ts

```diff
@@ -23,11 +23,13 @@ export namespace PhoneInput {
     pill: boolean;
     value: string;
     errorMessage: string;
+    flagClass: string;
   };
 
   @customElementWithCheck("md-phone-input")
   export class ELEMENT extends LitElement {
-    @property({ type: String }) codePlaceholder = "+1";
+    @property({ type: String, reflect: true }) codePlaceholder = "+1";
+    @property({ type: String }) flagClass = "flag__usa";
```

Comment:
consider this an element attribute, so be sure that the expected value input is know / intuitive to a user for a range of countries they may set this to, otherwise we should conceal it as an internal property.